### PR TITLE
Add timings to vnode_get/put stats

### PR DESCRIPTION
**Pull request by russelldb**
Add per partition stats for get/put. Test as with the other stat PRs. Do some gets / puts against a node, attach to it and run:

```
[{Stat, folsom_metrics:get_metric_value(Stat)} || 
 Stat <- folsom_metrics:get_metrics(), 
             is_tuple(Stat),
             element(1, Stat) == riak_kv,
             element(2, Stat) == vnode_gets].  
```

Or

```
[{Stat, folsom_metrics:get_metric_value(Stat)} || 
 Stat <- folsom_metrics:get_metrics(), 
             is_tuple(Stat),
             element(1, Stat) == riak_kv,
             element(2, Stat) == vnode_puts].  
```

### CODE REVIEW

**Comment by coderoshi** [comment]
+1

File: src/riak_kv_vnode.erl [hunk 1]
```diff
@@ -280,9 +280,11 @@ handle_command(?KV_PUT_REQ{bkey=BKey,
                            start_time=StartTime,
                            options=Options},
                Sender, State=#state{idx=Idx}) ->
+    StartTS = os:timestamp(),
     riak_kv_mapred_cache:eject(BKey),
```
Comment (beerriot) [inline src/riak_kv_vnode.erl:284]: A note to PR #433: merging with this commit may create a conflict that tries to resurrect the call to `riak_kv_mapred_cache`.

**Comment by beerriot** [comment]
It's quite interesting that per-vnode stats are not expunged when the vnode shuts down. I think that's a good thing, because you won't "just miss" a vnode's final stats after it hands off. And, given riak's on-demand vnode startup, "zero" spirals for non-running vnodes are technically correct.

It might be a good idea to head off some possible worry and questions by explaining this in our user docs, though: why some nodes show stats for all vnodes, others only some; why it's okay that many show zero forever; etc.

File: src/riak_kv_vnode.erl (hunk 1 above)
Comment (russelldb) [inline src/riak_kv_vnode.erl:284]: Noted, thank you.

**Comment by russelldb** [comment]
Good point. I don't believe we have _any_ docs about stats. The release notes is one place that this can be mentioned. I'll make sure it is (once this PR is merged.)

That aside, is this a +1?

**Comment by beerriot** [comment]
Sounds great, +1.

**Pull request merged**